.gitignore

```diff
@@ -1,5 +1,4 @@
 /.idea/
```

Comment:
`.idea` should be removed too. It's upon developer which tools to use as we already discussed.